SECURITY.md

```diff
@@ -0,0 +1,40 @@
+# Security Policy
+
+[//]: # (ATTENTION: This is the Markdown version of docs/security.rst. Any changes should also be made in the ReStructuredText version.)
+
```

Comment:
Why do we need two versions ?

Reply:
The convention for security notices is to have them in Markdown, but I couldn't find an easy way to load Markdown into Sphinx (without adding another library to the docs dependencies, like [sphinx-mdinclude](https://github.com/omnilib/sphinx-mdinclude)).

What do you think? Is it worth adding it in?

Reply:
Done.